README.md

```diff
@@ -33,28 +33,38 @@ First, be sure to register for an [airtable](https://airtable.com) account, crea
 ```ruby
 # Pass in api key to client
 @client = Airtable::Client.new("keyPCx5W")
+# Or if you have AIRTABLE_KEY varibale you can use
+@client = Airtable::Client.new
 ```
-
 Your API key carries the same privileges as your user account, so be sure to keep it secret!
 
+### Accessing a Base
+
+Now we can access any base in our Airsheet account by referencing the [API docs](https://airtable.com/api):
+
+```ruby
+# Pass in the base id
+@base = @client.table("appPo84QuCy2BPgLk")
+```
+
 ### Accessing a Table
 
 Now we can access any table in our Airsheet account by referencing the [API docs](https://airtable.com/api):
 
 ```ruby
-# Pass in the app key and table name
-@table = @client.table("appPo84QuCy2BPgLk", "Table Name")
+# Pass in the table name
+@table = @base.table("Table Name")
 ```
 
-### Querying Records
+### Batch Querying All Records
 
 Once you have access to a table from above, we can query a set of records in the table with:
 
 ```ruby
-@records = @table.records
+@records = @table.select
 ```
 
-We can specify a `sort` order, `limit`, and `offset` as part of our query:
+We can specify a `sort` order, `limit`, `max_records` and `offset` as part of our query:
```

Comment:
Maybe take out `max_records` from here since it can be confusing to see next to `limit` and most people don't need to use it. We can add a note about it below where we say "This executes a variable number of network requests (100 records per batch) to retrieve all records in a sheet."

instead could say:

"This executes a variable number of network requests (by default 100 records per batch) to retrieve all records in a sheet. You can use `max_records` to customize the number of records to fetch per batch."

Reply:
Then I think better to use `per_page` instead of `max_records`.